include/picongpu/param/speciesAttributes.param

```diff
@@ -162,6 +162,44 @@ namespace picongpu
         flylite::Superconfig::create( 0. )
     );
 
+    //atomic State Example
+    constexpr uint8_t numberLevels_Hydrogen = 4u;  // 4u =^= unsigned int 4
+
+    alias( atomicConfigNumber );
+    /** atomicConfigNumber
+     *
+     * must be defined for each ion seperatly by giving the number of moddeled
+     * atomic Levels N and the dataformat to be used, eg. float_x, uint32_t, ...
+     */
+
+    using ConfigNumber_H =
+        particles::atomicPhysics::stateRepresentation::ConfigNumber<
+            float_X,
+            numberLevels_Hydrogen
+        >;
+    //necessary since all "," are interpreted as seperating macro arguments
```

Comment:
For the most reader it would be not not clear where this comment belongs too.

Never the less I know what you mean, IMO you can write:

```
value_identifier(
    ConfigNumber_H,
    (
         particles::atomicPhysics::stateRepresentation::ConfigNumber<
             float_X,
             numberLevels_Hydrogen
         >
    ),
    ConfigNumber_H( 1u )
);
```

Reply:
clarified and moved comment.

Interestingly I got an error when trying to compile even with brackets in the value_identifier, therefore did not change the code structure.